package.json

```diff
@@ -81,6 +81,7 @@
     "style-loader": "^1.1.3",
     "stylelint": "^13.3.2",
     "stylelint-config-standard": "^20.0.0",
+    "stylelint-order": "^4.1.0",
```

Comment:
Do we expect to have changes in yarn.lock file? (not sure!)

Reply:
Yes, all package changes within `package.json` should result to `yarn.lock` changes. Please run `yarn install`.

Reply:
Done.